taxi/views.py

```diff
@@ -87,3 +96,38 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
```

Comment:
The `DriverLicenseUpdateForm` should only include fields that are intended to be updated. Ensure that the form is correctly configured to handle only the `license_number` field if that's the intended purpose.